lib/tmuxinator/project.rb

```diff
@@ -41,33 +41,17 @@ class Project
 
     def self.load(path, options = {})
       yaml = begin
-        raw_content = File.read(path)
-
-        args = options[:args] || []
-        @settings = parse_settings(args)
-        @args = args
-
-        content = Erubis::Eruby.new(raw_content).result(binding)
-        YAML.safe_load(content, [], [], true)
+        config_file_type = options[:type] || 'default'
+        # now must pass in type
+        # I temporarily hard coded it for getting 'procfile' type up and running
+        yml = Parser.new(path, options, 'procfile').build_yaml
```

Comment:
Obviously the 'procfile' type is temporarily hard coded. I didn't want to dive into cli changes until we get the parsing mechanism close to what we want.